patch_art.sh

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash
+
+ART_FILE=zmk/app/boards/shields/nice_view/widgets/art.c
+PERIPHERAL_STATUS_FILE=zmk/app/boards/shields/nice_view/widgets/peripheral_status.c
+
+reset_zmk() {
+    cd zmk
+
+    # Clean slate repo
+    git restore .
+    git clean -fd
+
+    cd ..
+}
+
+generate_lvgl_img() {
+    local name all_caps
+    name=$1
+    src=$2
```

Comment:
```suggestion
    local name="$1"
    local src="$2"
```

See https://www.shellcheck.net/wiki/SC2155 for why declaration and assignment is sometimes separate.